.github/workflows/build.yaml

```diff
@@ -23,28 +23,17 @@ on:
         type: string
 
 permissions:
-  packages: read
+  packages: write
   contents: read
 
 defaults:
   run:
     shell: bash
 
 jobs:
-  d:
-    name: Discover Platforms
-    runs-on: ubuntu-latest
-    outputs:
-      missing-platforms: ${{steps.discover.outputs.missing-platforms}}
-      p: ${{steps.discover.outputs.platforms}}
-    steps:
-      - name: Discover Platforms
-        id: discover
-        uses: AntelopeIO/discover-platforms-action@v1
-        with:
-          platform-file: .cicd/platforms.json
-          password: ${{secrets.GITHUB_TOKEN}}
-          package-name: builders
+  build-base:
+    uses: AntelopeIO/leap/.github/workflows/build_base.yaml@GH-1156-ph-cicd-nodeos-versions
```

Comment:
I think this should just be
```yaml
 uses: .github/workflows/build_base.yaml
```
Hardcoding `AntelopeIO` and/or `leap` breaks forks/mirrors, and we shouldn't be hardcoding the branch -- it should just use the same ref as the caller

Reply:
Addressed: 477f62c85779 and 0da3e7b36eb